### PR TITLE
[DOCS-5975] Mobile Application Testing Banner Updates

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Update beta banners + add site region specificity.

### Motivation

Chat with Mike on Slack, ready to merge.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/mobile_app_testing/_index.md [hunk 1]
```diff
@@ -24,8 +23,14 @@ cascade:
...
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSeHny7qHl5w3u3DCI4Ilc-r4IQZSAFOeZgMvP3CKBO9hEl1qA/viewform" >}}
-  Mobile Application Testing is in private beta. To request access, complete the form.
+{{% site-region region="us3,us5,eu,gov,ap1" %}}
+<div class="alert alert-warning">
+  Mobile Application Testing is only supported for Datadog customers using the US1 site.
+</div>
+{{% /site-region %}}
+
+{{< callout url="#" header="false" btn_hidden="true" >}}
+  Mobile Application Testing is in limited availability.
```
Comment (michaelperalta) [inline content/en/mobile_app_testing/_index.md:33]: Can we update this to include a note about it being in US1 only so customers don't have to switch the Datadog site to see that? 

Something like:

Mobile Application Testing is in Limited Availability for customers using the US1 site only.

**Review by michaelperalta — DISMISSED** [review]

**Review by rtrieu — CHANGES_REQUESTED** [review]
This could be a personal preference but I referenced phrasing we've used before in our docs that is slightly more concise.

**Review by rtrieu — APPROVED** [review]
Looks good!

File: content/en/mobile_app_testing/_index.md [hunk 2]
```diff
@@ -24,9 +23,13 @@ cascade:
     tags: ['mobile_testing']
 ---
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSeHny7qHl5w3u3DCI4Ilc-r4IQZSAFOeZgMvP3CKBO9hEl1qA/viewform" >}}
-  Mobile Application Testing is in private beta. To request access, complete the form.
-{{< /callout >}} 
+{{< site-region region="us3,us5,gov,eu,ap1" >}}
+<div class="alert alert-warning">Mobile Application Testing is not supported.</div>
```
Comment (kayayarai) [inline content/en/mobile_app_testing/_index.md:27]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
```

File: content/en/mobile_app_testing/mobile_app_tests/_index.md [hunk 3]
```diff
@@ -17,9 +16,13 @@ further_reading:
   text: "Best practices for maintaining end-to-end tests"
 ---
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSeHny7qHl5w3u3DCI4Ilc-r4IQZSAFOeZgMvP3CKBO9hEl1qA/viewform" >}}
-  Mobile Application Testing is in private beta. To request access, complete the form.
-{{< /callout >}} 
+{{< site-region region="us3,us5,gov,eu,ap1" >}}
+<div class="alert alert-warning">Mobile Application Testing is not supported.</div>
```
Comment (kayayarai) [inline content/en/mobile_app_testing/mobile_app_tests/_index.md:20]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
```

File: content/en/mobile_app_testing/mobile_app_tests/advanced_options.md [hunk 4]
```diff
@@ -20,9 +19,13 @@ further_reading:
   text: "Learn about Synthetic Monitoring Data Security"
 ---
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSeHny7qHl5w3u3DCI4Ilc-r4IQZSAFOeZgMvP3CKBO9hEl1qA/viewform" >}}
-  Mobile Application Testing is in private beta. To request access, complete the form.
-{{< /callout >}} 
+{{< site-region region="us3,us5,gov,eu,ap1" >}}
+<div class="alert alert-warning">Mobile Application Testing is not supported.</div>
```
Comment (kayayarai) [inline content/en/mobile_app_testing/mobile_app_tests/advanced_options.md:23]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
```

File: content/en/mobile_app_testing/mobile_app_tests/results.md [hunk 5]
```diff
@@ -14,9 +13,13 @@ further_reading:
   text: "Learn about the Events Explorer"
 ---
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSeHny7qHl5w3u3DCI4Ilc-r4IQZSAFOeZgMvP3CKBO9hEl1qA/viewform" >}}
-  Mobile Application Testing is in private beta. To request access, complete the form.
-{{< /callout >}} 
+{{< site-region region="us3,us5,gov,eu,ap1" >}}
+<div class="alert alert-warning">Mobile Application Testing is not supported.</div>
```
Comment (kayayarai) [inline content/en/mobile_app_testing/mobile_app_tests/results.md:17]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
```

File: content/en/mobile_app_testing/mobile_app_tests/steps.md [hunk 6]
```diff
@@ -14,9 +13,14 @@ further_reading:
   text: "Learn about advanced options in mobile tests"
 ---
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSeHny7qHl5w3u3DCI4Ilc-r4IQZSAFOeZgMvP3CKBO9hEl1qA/viewform" >}}
-  Mobile Application Testing is in private beta. To request access, complete the form.
-{{< /callout >}} 
+{{< site-region region="us3,us5,gov,eu,ap1" >}}
+<div class="alert alert-warning">Mobile Application Testing is not supported.</div>
```
Comment (kayayarai) [inline content/en/mobile_app_testing/mobile_app_tests/steps.md:17]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
```

File: content/en/mobile_app_testing/settings/_index.md [hunk 7]
```diff
@@ -14,10 +14,13 @@ further_reading:
 ---
 {{< jqmath-vanilla >}}
 
-{{< callout url="https://docs.google.com/forms/d/e/1FAIpQLSeHny7qHl5w3u3DCI4Ilc-r4IQZSAFOeZgMvP3CKBO9hEl1qA/viewform" >}}
-  Mobile Application Testing is in private beta. To request access, complete the form.
-{{< /callout >}} 
+{{< site-region region="us3,us5,gov,eu,ap1" >}}
+<div class="alert alert-warning">Mobile Application Testing is not supported.</div>
```
Comment (kayayarai) [inline content/en/mobile_app_testing/settings/_index.md:18]:
```suggestion
<div class="alert alert-warning">Mobile Application Testing is not supported on this site.</div>
```

**Review by kayayarai — COMMENTED** [review]
Suggesting giving the user some context about why it's not supported when we show the site region message.

**Pull request merged**